File: java/Example3.java
// import anypackage.in.which.you.put.Exevalator;

/**
 * An example to use a variable.
 */
public class Example3 {

	/**
	 * The entry point of this example code.
	 * @param args Command-line arguments (unused)
	 */
	public static void main(String[] args) {

		// Create an instance of Exevalator Engine
		Exevalator exevalator = new Exevalator();

		// Declare a variable and set the value
		exevalator.declareVariable("x");
		exevalator.writeVariable("x", 1.25);

		// Evaluate the value of an expression
		double result = exevalator.eval("x + 1");

		// Display the result
		System.out.println("Result: " + result);
	}
}

File: python/example3.py
from exevalator import Exevalator

def main() -> None:
    """
    An example to use a variable.
    """

    # Create an instance of Exevalator Engine
    ex = Exevalator()

    # Declare a variable and set the value
    ex.declare_variable("x")
    ex.write_variable("x", 1.25)

    # Evaluate the value of an expression
    result = ex.eval("x + 1")

    # Display the result
    print(f"result: {result}")

if __name__ == "__main__":
    main()
